graphistry/pygraphistry.py

```diff
@@ -382,6 +377,11 @@ def refresh(token=None, fail_silent=False):
                 PyGraphistry._is_authenticated = True
             return PyGraphistry.api_token()
         except Exception as e:
+            if PyGraphistry.store_token_creds_in_memory():
```

Comment:
on `if` of 375 if statement, should we add an `else: return token` ?

Reply:
@vaimdev how about the comment ^^^ on missing `else: return token` ? a bug to fix or correct as is?

The implicit `else` case currently ignores `token`, returning instead stale, right?